book/SUMMARY.md

```diff
@@ -31,6 +23,17 @@
 
 # Zapp Plugins
 
+## Zapp Plugin Abilities
+  - [Startup Hooks](/plugins/general-abilities/startup-hooks/startup-hooks.md)
+    - [iOS](/plugins/general-abilities/startup-hooks/ios/startup-hooks-ios.md)
+    - [Android](/plugins/general-abilities/startup-hooks/android/startup-hooks-android.md)
+  - [JS2Native and Events](/plugins/general-abilities/js2native/readme.md)
+  - [Communication Between Plugins](/plugins/general-abilities/communication-between-plugins/communication-between-plugins.md)
+  - [Screen Hook](/plugins/general-abilities/screen-hooks/screen-hooks-general.md)
+    - [iOS](/plugins/general-abilities/screen-hooks/ios/screen-hook-ios.md)
+    - [Android](/plugins/general-abilities/screen-hooks/android/screen-hook-android.md)
+  - [One Code Base for Multiple Plugins](book/plugins/general-abilities/multiple-types-one-repo/multiple-types-one-repo.md)
```

Comment:
First of all - this is all about multiple plugin TYPES - hence shouldn't be in abilities
Plus I feel it should be somewhere in the intro area and not in here

Reply:
Moved it to `Deploy and Submit` section